Makefile

```
@@ -307,20 +309,20 @@ bpf-test: generate ginkgo-set ## Run BPF tests.$(GOBIN)
 	$(SUDO) $(ENVTEST_ASSETS_DIR)/ginkgo \
 		./pkg/bpftest/bpftest.test
 
-escapes_detect: tidy-vendor
+escapes_detect: tidy-vendor ## Tidy Go dependencies.
```

```suggestion
escapes_detect: tidy-vendor 
```

I would move the description you added to the tidy-vendor target